### ISSUE TITLE
Custom Range Doesn't Work

### ISSUE THREAD

**Issue by aldmbmtl** [juno-fx/report#104]
I passed in 4 comma separated frame ranges to the submitter and rendered the min and max frame in the range.

**Comment by skeleturtle** [comment]
I will need a better understanding of the expected results for 4 separate frame ranges from the same write node. From deadline job to Juno file creation.

**Comment by skeleturtle** [comment]
Just giving this a little bump, as I still need confirmation on the expected result.

**Comment by aldmbmtl** [comment]
The standard behavior in Nuke is I can render a comma separated list of frames or frame ranges. It will all render to a single Juno file, it will only have certain frames though. This is super important when we are doing only parts of a single huge sequence. For example, the shot that I was doing had 2k frames but I could split it up into 3 parts. I had to precomp this out into 3 Juno files when in reality, it could have been just one Juno file with just the frames I needed. The frames in question are smart vectors which are data channels and I don't need them rendered on the rest of the range I'm not working on.

**Comment by skeleturtle** [comment]
ah okay, so it would be a non-sequential render within a frame range, that would all be associated to a single juno file.

I'll look into how to manage this. I'm not sure the best way to handle the first/last frame on the juno file other than just doing what we currently, and maybe just adding a key to the metadata that it's non-sequential?

**Comment by aldmbmtl** [comment]
I'll be honest, just let it be the full frame range. Then if we import it, we would just need to set the read node to "black" on the "on error" knob and that will do exactly what we want.

**Comment by skeleturtle** [comment]
yup, by adding the metadata that it's non-sequential I was going to trigger the black on error.

**Comment by aldmbmtl** [comment]
A man after my own heart...

**Comment by skeleturtle** [comment]
builds are prepped for release
